qt-models/messagehandlermodel.cpp

```diff
@@ -51,6 +51,15 @@ void MessageHandlerModel::addLog(QtMsgType type, const QString& message)
 #endif
 }
 
+void MessageHandlerModel::copyLogToString(QString &copyString)
```

Comment:
Please return the `QString`, don't use an out-parameter.

Reply:
Yeah, idiomatically I'd prefer that as well